command_server/command_server.py

```diff
@@ -37,7 +38,9 @@ def __init__(self, host=None, port=None, loop=None):
         logging.info('Init Server on host %s:%s' % (host, port))
         self._loop = loop or asyncio.get_event_loop()
         #TODO: MOve debug mode to config file.
-        self._server = self._loop.create_server(lambda: HttpRequestHandler(),
+        self._app = web.Application(loop=loop)
```

Comment:
You don't need  handle_request in this case.
Have you checked this code. It's work?
In your implementation, we can't track request in general, and HttpRequestHandler  lost the main sense.
I am not sure that using  web.Application it's the best solution.